cli/templates/plain_request_urls_list/main.js

```diff
@@ -0,0 +1,53 @@
+const Apify = require('apify');
```

Comment:
I'd add comments to all functions.

Also I'd make all examples as simple as possible - ie. remove that interval with logs, unneeded parameters as minConcurrency or maxOpenPagesPerInstance...